README.md

```diff
@@ -72,7 +72,7 @@ export RESOURCE_GROUP_NAME=give_your_name_citus_test_automation_r_g
 # export REGION=eastus2
 
 # open and modify the instance types/discs as you wish
-less azuredeploy.parameters.json
+vim azuredeploy.parameters.json
```

Comment:
It is possible that `vim` is not installed in some people whereas `less` will exist. The point here was to show to parameters file and if people want they can change it with their favorite editor.

Reply:
that makes sense, reverting this commit